packages/x-charts-pro/src/FunnelChart/FunnelPlot.tsx

```diff
@@ -20,6 +20,11 @@ export interface FunnelPlotProps extends FunnelPlotSlotExtension {
    * @default 0
    */
   gap?: number;
+  /**
+   * The radius, in pixels, of the corners of the funnel sections.
+   * @default 0
```

Comment:
@noraleonte Should we make the default >0? Maybe also the gap? The funnel with border radius look quite nice to me 😄

Reply:
Yes, I think we should. We can make it an 8 by default for a softer look 💙